.templates/openstack/properties.yml

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  shield:
+    daemon_public_key: (( vault "secret/dc01/proto/shield/keys/core:public" ))
+
+properties:
+  prometheus:
+    rule_files: 
+    - /var/vcap/packages/bosh_alerts/*.alerts
+    - /var/vcap/packages/cloudfoundry_alerts/*.alerts
+    scrape_configs:
+    - (( append ))
+    - job_name: bosh_graphite
+      static_configs:
+      - targets: (( grab meta.graphite_exporter_targets ))
+    - job_name: firehose
+      static_configs:
+      - targets: (( grab meta.firehose_exporter_targets ))
+    - job_name: cf
+      scrape_timeout: 60s
+      static_configs:
+      - targets: (( grab meta.cf_exporter_targets ))
+    web:
+      external_url: (( concat "http://" jobs.nginx.networks.vip.static_ips.0 ":9093" ))
+
+  alertmanager:
+    # Slack Alerts can be configured here, put the slack webhook in vault
+    #receivers:
+    #- name: default-receiver
+    #  slack_configs:
+    #  - api_url: (( vault "secret/aws-clients/shared/slack:webhook" ))
+    #    channel: (( concat "#" meta.env "-alerts" ))
+    web:
+      external_url: (( concat "http://" jobs.nginx.networks.vip.static_ips.0 ":9093" ))
+
+  grafana: 
+    server:
+      root_url: (( concat "http://" jobs.nginx.networks.vip.static_ips.0 ":9093" ))
+    dashboards: 
+      json:
+        enabled: true
+    prometheus: 
+      dashboard_files: 
```

Comment:
If you enable the `dashboard_files` by default, then you need to add the `bosh_dashboards`,`cloudfoundry_dashboards` and `system_dashboards` to the job templates, otherwise the grafana post-start script will fail because those dashboard don't exists.